backend/src/main/resources/hibernate.properties

```diff
@@ -1 +1,2 @@
 hibernate.types.print.banner=false
+hibernate.jdbc.time_zone=UTC
```

Comment:
Possibly this should live in the `dev` profile instead, since it's such a local-setup issue? It seems like a no-brainer in general, though.